site/en/reference/cli/concepts/netdata.md

```diff
@@ -29,6 +32,19 @@ unpublish
 Done
 ```
 
+### `full` commands
+
+The `full` commands report the flag status or resest the flag tracking whether
```

Comment:
resest = reset

Can we rephrase the whole thing though cause it sounds a little awkward.  Maybe:

The `full` commands are used for the flag that tracks whether the "net data full" callback has been invoked.  These commands can report the flag's status or reset it.

Reply:
This still needs to be addressed